### PR TITLE
Add functionality Force

### PR DESCRIPTION
**Pull request by HanB1n**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AssignWeddingCommand.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        if (index.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        }
+
+        Person personToEdit = lastShownList.get(index.getZeroBased());
+
+        for (Wedding wedding : weddingsToAdd) {
+            if (!model.hasWedding(wedding)) {
+                if (this.force) {
+                    CreateWeddingCommand newWedding = new CreateWeddingCommand(wedding);
```
Comment (dasha3412) [inline src/main/java/seedu/address/logic/commands/AssignWeddingCommand.java:85]: Maybe can make this variable 'newWeddingCommand' to make the code clearer

File: src/main/java/seedu/address/logic/commands/DeleteTagCommand.java [hunk 2]
```diff
@@ -11,10 +11,9 @@
 import seedu.address.model.tag.Tag;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using its tag name in the Wedlinker.
```
Comment (dasha3412) [inline src/main/java/seedu/address/logic/commands/DeleteTagCommand.java:14]: Maybe can say 'Deletes a person identified using their displayed index in the Wedlinker and the tag name'

**Review by dasha3412 — COMMENTED** [review]
LGTM, some minor comment/variable changes

File: src/main/java/seedu/address/logic/commands/DeleteWeddingCommand.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+        for (Wedding wedding : allWeddings) {
+            if (wedding.getWeddingName().equals(targetWedding.getWeddingName())) {
+                if (wedding.canBeDeleted()) {
+                    model.deleteWedding(wedding);
+                    return new CommandResult(String.format(MESSAGE_DELETE_WEDDING_SUCCESS,
+                            Messages.format(targetWedding)));
+                } else {
+                    throw new CommandException(
+                            String.format(MESSAGE_DELETE_WEDDING_FAILURE_STILL_USED, Messages.format(targetWedding)));
```
Comment (riccoljy) [inline src/main/java/seedu/address/logic/commands/DeleteWeddingCommand.java:49]: CodeCov flagged this as not covered -- also couldn't find any tests that had `MESSAGE_DELETE_WEDDING_FAILURE_STILL_USED` as output, but maybe I overlooked!

**Review by riccoljy — APPROVED** [review]
LGTM besides some CodeCov

File: src/main/java/seedu/address/logic/commands/AssignWeddingCommand.java (hunk 1 above)
Comment (HanB1n) [reply src/main/java/seedu/address/logic/commands/AssignWeddingCommand.java:87]: Fixed.

**Pull request merged**